easybuild/easyconfigs/l/libbeef/libbeef-0.1.3-GCCcore-10.3.0.eb

```diff
@@ -0,0 +1,35 @@
+# #
+# This is a contribution from HPCNow! (http://hpcnow.com)
+# Copyright::   HPCNow!
+# Authors::     Aurelio Rodriguez <aurelio.rodriguez@hpcnow.com>
+# License::     GPL-v3.0
+# #
+easyblock = 'ConfigureMake'
+
+name = 'libbeef'
+version = '0.1.3'
+local_commit = '188e440'
+
+homepage = 'https://github.com/vossjo/libbeef'
+description = """
+libbeef is a package providing the class of BEEF-vdW functionals
+and Bayesian error estimates to Fortran- or C-based DFT codes.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '10.3.0'}
```

Comment:
I believe we should be using GCC level for this, as i think it could be performance difference and one might want to keep a separate version with intel compilers.